Adding Rust Hack and Learn Videos to Appearance (#2596)
mainmatter/mainmatter.com: src/appearances/2024-11-28-hack-and-learn-hamburg-marco.md

---

> title: "A blueprint for Rust / Axum apps"

Reviewer: ```suggestion
title: "A blueprint for Rust/Axum apps"
```

please change image accordingly

image: "/assets/images/talks/2024-11-28-hack-and-learn-hamburg/thumbnail-marco.png"
url: https://youtu.be/5GM_ZubHyIk
media: video
channel: rust-hackandlearn-hamburg
---

> Marco Otte-Witte introduced Gerust's architecture and tooling for simplifying Rust backend projects at Rust Hack & Learn in Hamburg.

Reviewer: ```suggestion
Marco Otte-Witte introduced [Gerust](https://gerust.rs), a generator for Rust backend projects at Rust Hack & Learn in Hamburg.
```
